myproject/urls.py

```diff
@@ -16,11 +16,17 @@
 """
 from django.contrib import admin
 from django.urls import include, path
-from django.views.generic import RedirectView
+from rest_framework_simplejwt import views as jwt_views
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("helloWorld/", include("helloWorldApp.urls")),
-    path("", RedirectView.as_view(url="helloWorld/", permanent=True)),
     path("api-auth/", include("rest_framework.urls")),
+    path(
+        "api/token/", jwt_views.TokenObtainPairView.as_view(), name="token_obtain_pair"
+    ),
+    path(
+        "api/token/refresh/", jwt_views.TokenRefreshView.as_view(), name="token_refresh"
+    ),
+    path("bikelist/", include("bikeLists.urls")),
```

Comment:
Rename bikelist to bikes. Nouns are preferred for endpoints. Rename the model as well to Bikes or Bike